common/traits/wc_genetic_traits.txt

```diff
@@ -224,4 +226,51 @@ magic_good_noninheritable = {
 			desc = trait_magic_good_noninheritable_character_desc
 		}
 	}
-}
\ No newline at end of file
+}
+
+dire = {
+	index = 2508
+	opposites = {
+		weak
+		physique_bad
+		dwarf
+		giant
+	}
+	
+	potential = {
+		OR = {
+			dire_possible_trigger = yes
+			mother = { dire_possible_trigger = yes }
+			real_father = { dire_possible_trigger = yes }
+		}
+	}
+	
+	birth = 0.5
+	random_creation = 0.5
```

Comment:
It makes it so rare there are no dire trolls on Zandalar.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/115731229-050c9800-a398-11eb-94a0-b03654e0677d.png)

</details>

I would change it back to 5.

Reply:
> I would change it back to 5.

Changed in bd0dca9def81c5917ab248912b6b2b41613d2fae. 👻